packages/flame/lib/src/sprite.dart

```diff
@@ -62,6 +62,21 @@ class Sprite {
     src = (position ?? Vector2.zero()).toPositionedRect(srcSize);
   }
 
+  /// Returns a new [Sprite] where the image in memory is just the region
+  /// defined by the original sprite.
+  ///
+  /// **Note:** This is a heavy async operation and should not be called inside
+  /// the game loop. Remember to call dispose on the image object once you
+  /// aren't going to use it anymore.
```

Comment:
Since it is async it shouldn't matter right? As long as it isn't called every tick.

Reply:
Yeah, I kinda used it here to keep consistent with the other methods, should I remove on this one?

Reply:
Yeah, I think it should be fine to remove it, or rewrite it to explain that it shouldn't be called every tick.